src/Symfony/Component/Validator/README.md

```diff
@@ -7,7 +7,7 @@ The Validator component provides tools to validate values following the
 Resources
 ---------
 
-  * [Documentation](https://symfony.com/doc/current/book/validation.html)
+  * [Documentation](https://symfony.com/doc/current/components/validator.html)
```

Comment:
@nicolas-grekas is right. I guess we just linked to the book chapter here before because the Validator component docs were added a long time after the book chapter and we hadn't anything else to link to. Thus, your change is actually perfect and I was wrong @CoolGoose. :) Thank you!